src/test/e2e/manage/roles/roles-overview.admin.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { faker } from '@faker-js/faker';
+import { expect, test } from '@playwright/test';
+
+test.describe('Roles Overview - Admin', () => {
+  test.beforeEach(async ({ page, context }) => {
+    await context.addCookies([
+      {
+        name: 'ff_react_roles_overview',
+        value: 'true',
+        domain: 'localhost',
+        path: '/',
+      },
+    ]);
+    await page.goto('/manage/roles');
+  });
+
+  test('can navigate to roles overview page', async ({ page }) => {
+    await expect(page.getByRole('heading', { level: 1 })).toContainText(
+      'Rollen',
+    );
+  });
+
+  test('can search for roles', async ({ page }) => {
+    const initialRows = page.getByRole('row');
+    const initialFirstRow = await initialRows.nth(1).textContent();
+
+    const searchInput = page.getByLabel('Zoeken');
+    await searchInput.fill('e2e');
+    await page.waitForLoadState('networkidle');
+    await expect(searchInput).toHaveValue('e2e');
+    const e2eRows = page.getByRole('row');
+    const e2eResultRow = e2eRows.nth(1).filter({
+      hasText: 'e2e',
+    });
+    await expect(e2eResultRow).toBeVisible({ timeout: 8_000 });
```

Comment:
8s seems like a rather long timeout.
Is there an other way? 
With `waitForLoadState` ?